<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-26T14:42:30Z" name="Default suite" finished-at="2023-01-26T14:42:33Z" duration-ms="2834">
    <groups>
    </groups>
    <test started-at="2023-01-26T14:42:30Z" name="Default test" finished-at="2023-01-26T14:42:33Z" duration-ms="2834">
      <class name="testcase.adminButtontest">
        <test-method is-config="true" signature="main()[pri:0, instance:testcase.adminButtontest@36060e]" started-at="2023-01-26T14:42:26Z" name="main" finished-at="2023-01-26T14:42:30Z" duration-ms="4042" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main -->
        <test-method signature="adminButton()[pri:0, instance:testcase.adminButtontest@36060e]" started-at="2023-01-26T14:42:30Z" name="adminButton" finished-at="2023-01-26T14:42:33Z" duration-ms="2802" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [18] but found [20],
expected [(255, 255, 255)] but found [rgb(255, 255, 255)]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [18] but found [20],
expected [(255, 255, 255)] but found [rgb(255, 255, 255)]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at testcase.adminButtontest.adminButton(adminButtontest.java:95)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- adminButton -->
      </class> <!-- testcase.adminButtontest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
